src/components/Editor.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef } from "react";
+import EmailEditor, { UnlayerOptions, User } from "react-email-editor";
+import mergeTags from "../external/merge.tags";
+
+export interface EditorProps {
```

Comment:
Exporting this type is not necessary, right?

Reply:
Yes right, not now.